Mathlib/MeasureTheory/Group/Measure.lean

```diff
@@ -246,6 +248,27 @@ end MeasurableMul
 
 end Mul
 
+section Semigroup
+
+variable [Semigroup G] [MeasurableMul G]
+
+/-- The image of a left invariant measure under scalar multiplication is left invariant. -/
+@[to_additive]
+theorem isMulLeftInvariant_map_smul
+    {α} [SMul α G] [SMulCommClass α G G] [MeasurableSpace α] [MeasurableSMul α G]
+    [IsMulLeftInvariant μ] (a : α) :
+    IsMulLeftInvariant (map (a • · : G → G) μ) :=
+  (forall_measure_preimage_mul_iff _).1 <| fun x _ hs =>
+    (SMulInvariantMeasure_map_smul μ a).measure_preimage_smul x hs
+
+/-- The image of a left invariant measure under right multiplication is left invariant. -/
+@[to_additive isMulLeftInvariant_radd]
+theorem isMulLeftInvariant_rmul [IsMulLeftInvariant μ] (g : G) :
```

Comment:
Should this be an `instance`?

Reply:
The other similar theorems around here aren't instances, so I am not sure